src/Symfony/Component/HttpKernel/Profiler/FileProfilerStorage.php

```diff
@@ -18,6 +18,8 @@
  */
 class FileProfilerStorage implements ProfilerStorageInterface
 {
+    protected const MAX_RETENTION_DAYS = 1;
```

Comment:
I would move this const and hardcode 1 day in the private method. I don't see any use cases where you would need to look at an older profile.